README.md

```diff
@@ -15,18 +15,33 @@ Eclipse Kura™
   
 </div>
 
-**Eclipse Kura**, from [the maori word for tank/container](https://maoridictionary.co.nz/search/?keywords=kura), is an OSGi-based Application Framework for M2M Service Gateways
-
-Kura aims at offering a Java/OSGi-based container for M2M applications running in service gateways. Kura provides or, when available, aggregates open source implementations for the most common services needed by M2M applications. Kura components are designed as configurable OSGi Declarative Service exposing service API and raising events. While several Kura components are in pure Java, others are invoked through JNI and have a dependency on the Linux operating system.
-
-For more information, see the [Eclipse project proposal](http://www.eclipse.org/proposals/technology.kura/).
+## What is Eclipse Kura™?
+From [the maori word for tank/container](https://maoridictionary.co.nz/search/?keywords=kura), Eclipse Kura™ is a versatile software framework designed to supercharge your edge devices. With an intuitive web interface, Eclipse Kura™ streamlines the process of configuring your gateway, connecting sensors, and IoT devices to seamlessly collect, process, and send data to the cloud. Eclipse Kura™ provides an extensible Java API for developing custom plugins within the framework. Additionally, it offers a REST API, enabling the use of Eclipse Kura™ as a backend service in your application.
+ 
+Eclipse Kura™ runs on an edge gateway, which can be anything from a small SBC(single-board computer) like a Raspberry Pi, or a powerful high-performance computer.
+
+## What can Eclipse Kura™ do for me?
```

Comment:
I would set this as an H3 or H4 header just so it's clear that the first section of the readme is dedicated to explaining what is Kura

```suggestion
### What can Eclipse Kura™ do for me?
```